ui/src/screen/project.ts

```diff
@@ -0,0 +1,172 @@
+import { get, derived, Readable } from "svelte/store";
+
+import type { PeerId } from "../identity";
+import * as project from "../project";
+import * as remote from "../remote";
+import type { Urn } from "../urn";
+import * as validation from "../validation";
+
```

Comment:
There seems to be a mix of remote modal and peer selector logic in here, which I found a bit confusing when reviewing the code. Could we further split the functionality in separate modules?

Reply:
Could you expand on what confuses you?

Reply:
It's hard for me to determine the responsibilities of the functions. Which UI features are they used for kind of thing.

Does that make sense?

Reply:
Yes and no, the stores in this file are highly interdepent and are exclusive to the `Screen/Project.svelte` and its `Modal/ManagePeers.svelte`.

Reply:
Ok, well then ignore me :)